core-api/src/main/java/org/codice/imaging/nitf/core/image/ImageSegment.java

```diff
@@ -15,9 +15,11 @@
 package org.codice.imaging.nitf.core.image;
 
 import java.util.List;
+
 import javax.imageio.stream.ImageInputStream;
-import org.codice.imaging.nitf.core.common.CommonBasicSegment;
+
 import org.codice.imaging.nitf.core.common.DateTime;
+import org.codice.imaging.nitf.core.common.CommonBasicSegment;
 
 /**
  Image segment information.
```

Comment:
I'm not sure whether you're asking about the import or the class-level javadoc, but we need both.  Without the javadoc, checkstyle will complain. Without the new import, methods like 'getHeaderLength()' won't be part of the interface.